toc/rfc/rfc-000-template.md

```diff
@@ -0,0 +1,14 @@
+# My RFC Title
```

Comment:
Should we have a metadata section like e.g. https://github.com/buildpacks/rfcs/blob/main/0000-template.md?

```md
# Meta
[meta]: #meta
- Name: (fill in the feature name: My Feature)
- Start Date: (fill in today's date: YYYY-MM-DD)
- Author(s): (Github usernames)
- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
- RFC Pull Request: (leave blank)
- CNB Pull Request: (leave blank)
- CNB Issue: (leave blank)
- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
```